content/bd/0004-relational-model-conversion.md

```diff
@@ -322,17 +321,17 @@ Existem também casos especiais para quando temos [participação obrigatória](
 - **_One-to-Many_ com participação obrigatória**: deixamos de precisar de uma nova relação,
   e colocamos os atributos na relação da entidade com multiplicidade 1.
 
+Nos exemplos seguintes, os atributos das entidades nas representações em modelo E-A serão omitidos por brevidade.
```

Comment:
```suggestion
Nos exemplos abaixo, os atributos das entidades nas representações em modelo E-A são omitidos por brevidade.
```
Tomayto tomahto, but it probably sounds better to refer to the text as a constant rather than a developing story.